e2e/cypress/integration/integrations/message_to_channel_via_slash_command_spec.js

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @integrations
+
+/**
+* Note: This test requires webhook server running. Initiate `npm run start:webhook` to start.
+*/
+
+describe('Integrations', () => {
+    let team;
+    let offTopicChannel;
+
+    before(() => {
+        cy.requireWebhookServer();
+
+        // # Login as sysadmin
+        cy.apiLogin('sysadmin');
+
+        // # Create new team and get off-topic channel
+        cy.apiCreateTeam('test-team', 'Test Team').then((teamResponse) => {
+            team = teamResponse.body;
+
+            cy.apiGetChannelByName(team.name, 'off-topic').then((res) => {
+                offTopicChannel = res.body;
+            });
+        });
+    });
+
+    beforeEach(() => {
+        // # Visit town-square
+        cy.visit(`/${team.name}/town-square`);
+    });
+
+    it('MM-T706 Error Handling for Slash Commands', () => {
```

Comment:
`MM-T706` will be used to integrate between Cypress and TM4J.  Will send separate PR to document at https://developers.mattermost.com/contribute/webapp/end-to-end-tests/